### ISSUE TITLE
React hooks

### ISSUE THREAD

**Issue by EladBezalel** [jchiatt/react-chaos#11]
Looks like a really nice project!

I think having a `useChaos` hook can be really useful and will also won't encapsulate the wrapped component

**Comment by jchiatt** [comment]
@EladBezalel I agree, this is a good idea. I'll add it to the roadmap. Thanks for sharing!

**Comment by darotar** [comment]
@EladBezalel @jchiatt  It seems reasonable because of hooks using everywhere, but I can't imagine how I would implement chaos with hooks in my project. Because with HOC, we can change logic of the whole child component, but can we do this with hook? And will it be clear enough to not write boilerplate condition for render every time?

Honestly, I can't even imagine about using this with renderProps pattern because we don't need any returnable value from chaos
